handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -198,4 +216,92 @@ static long memoryAddress(ByteBuf buf) {
     }
 
     private OpenSsl() { }
+
+    private static String normalizeOs(String value) {
```

Comment:
be nice to be able to share code with [os-maven-plugin](https://github.com/trustin/os-maven-plugin/blob/master/src/main/java/kr/motd/maven/os/Detector.java#L131) somehow, or at least have more formal definitions for this mapping. I guess we have an implicit dependency on os-maven-plugin generating the resource now?